internal/rfc7591/types_test.go

```diff
@@ -0,0 +1,49 @@
+package rfc7591v1_test
+
+import (
+	"testing"
+
+	"github.com/bufbuild/protovalidate-go"
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/testing/protocmp"
+
+	rfc7591 "github.com/pomerium/pomerium/internal/rfc7591"
+)
+
+func TestValidation(t *testing.T) {
+	v := &rfc7591.JsonWebKey{Kty: "Invalid"}
+	require.ErrorContains(t, protovalidate.Validate(v), `kty: value must be in list ["RSA", "EC", "oct", "OKP"] [string.in]`)
+}
+
```

Comment:
the purpose of the test was to ensure that protovalidate is properly wired, not to validate all conditions